src/main/java/com/example/controller/AuthController.java

```diff
@@ -15,7 +15,7 @@
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping(" /api/auth/")
+@RequestMapping("/api/auth/")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/auth/")
```